src/meta/processors/configMan/SetConfigProcessor.cpp

```diff
@@ -14,7 +14,7 @@ std::unordered_set<std::string> SetConfigProcessor::mutableFields_ = {
     // rocksdb_column_family_options
     "disable_auto_compactions",
     // TODO: write_buffer_size will cause rocksdb crash
-    // "write_buffer_size",
+    "write_buffer_size",
```

Comment:
"write_buffer_size" works well now ? if yes , do we need to uncomment in `MetaServerBasedPartManager::onSpaceOptionUpdated`?

Reply:
> "write_buffer_size" works well now ? if yes , do we need to uncomment in `MetaServerBasedPartManager::onSpaceOptionUpdated`?

Sorry, the comment has been removed. I'm wrong.

Reply:
Forgot to remove the TODO : )